### PR TITLE
Feature/154 specify platforms on project level

### LINKED ISSUE #154: Specify the release platforms on the project level
## Problem

We want to store the release platforms in the 'release_config.yml' file of the projects.
If the platforms in the user input are not provided, we will use the platforms from the config file.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #154

### CODE REVIEW

File: src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java [hunk 1]
```diff
@@ -24,38 +32,96 @@
...
-    }
-
     /**
      * Main entry point for all Release Droid's calls.
      */
     // [impl->dsn~rd-creates-validation-report~1]
     // [impl->dsn~rd-creates-release-report~1]
     public void run(final UserInput userInput) {
+        final Repository repository = this.repositoryGateway.getRepository(userInput);
+        validateUserInput(userInput, repository);
```
Comment (chiaradiamarcelo) [inline src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java:59]: I find it a bit confusing that this method is not only validating, but it's also changing the `UserInput`.

Maybe we can separate this "command" part to another method specific for it?

like `List<PlatformName> platformNames = getPlatformNames(userInput, repository.getPlatformNames());`

or something similar, and then pass that value to the use cases?

File: src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java [hunk 2]
```diff
@@ -56,23 +56,38 @@ public ReleaseDroid(final RepositoryGateway repositoryGateway, final ValidateUse
     // [impl->dsn~rd-creates-release-report~1]
     public void run(final UserInput userInput) {
         final Repository repository = this.repositoryGateway.getRepository(userInput);
-        validateUserInput(userInput, repository);
+        final List<PlatformName> platformNames = getPlatformNames(userInput, repository);
+        validatePlatformNames(platformNames);
+        validateUserInput(userInput);
```
Comment (chiaradiamarcelo) [inline src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java:61]: Maybe we can move this line to the beginning of the method?

**Review by chiaradiamarcelo — DISMISSED** [review]

File: src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java:61]: Tried this out, this makes testing more difficult. I will leave it be for now

Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java]: Okay, decided to move it as you suggested

**Pull request merged**